.github/workflows/ci.yml

```diff
@@ -38,6 +38,8 @@ jobs:
           git diff-index --quiet HEAD --
       - name: Build project
         run: make build
+      - name: Install to local repository
+        run: make releaselocal
```

Comment:
Added this to CI step so we'll know if the release build is likely to fail. The first commit to this PR only had this change (and you can see it failed the build).

Reply:
Can you make the `name` clearer to indicate intent? It's not obvious to me based on the description there now